public/css/drag-drop-style.css

```
@@ -0,0 +1,523 @@
+html.assessment-survey-standalone,
+body.assessment-survey-standalone {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: BalooBhai2, Kalam, sans-serif;
```

Stylelint (17.10.0)

[error] Expected quotes around "BalooBhai2" (font-family-name-quotes)

(font-family-name-quotes)